.goreleaser.yml

```diff
@@ -66,3 +66,4 @@ release:
   extra_files:
     - glob: ./build/out/crds.yaml
     - glob: ./deploy/manifests/nginx-gateway.yaml
+    - glob: ./deploy/manifests/nginx-plus-gateway.yaml
```

Comment:
question: does this determine what files are released? or runs pipeline job?

Reply:
@sjberman looks like we are also missing the new experimental manifests here